tests/keyword_test/keyword_test/keywords_in_table_generated.rs

```diff
@@ -22,7 +23,7 @@ impl<'a> flatbuffers::Follow<'a> for KeywordsInTable<'a> {
 impl<'a> KeywordsInTable<'a> {
   pub const VT_IS: flatbuffers::VOffsetT = 4;
   pub const VT_PRIVATE: flatbuffers::VOffsetT = 6;
-  pub const VT_TYPE_: flatbuffers::VOffsetT = 8;
+  pub const VT_TYPE: flatbuffers::VOffsetT = 8;
```

Comment:
Is this an intentional change? If so why? and maybe it should be part of a different PR.

Reply:
Not intentional; coming out!